public/search.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="stylesheets/css/reset.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/gh/kognise/water.css@latest/dist/dark.min.css"
+    />
+
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
+      integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
+      crossorigin="anonymous"
+    />
+    <link href="stylesheets/css/style.css" rel="stylesheet" />
+
+    <title>Search</title>
+  </head>
+
+  <header class="navbar">
+    <img
+      class="nav-logo-image"
+      src="img/dumbbell.png"
+      alt="fitness-buddy-logo"
+    />
+    <h2>Fitness Buddy</h2>
+  </header>
+  <body>
+    <div id="container-margin-nav">
+      <form id="signup-form" class="signup">
+        <h2 class="page-title">Search Exercise</h2>
+
+        <br />
+
+        <input
+          type="text"
+          id="exercise-name-input"
```

Comment:
```suggestion
          id="exercise-title-input"
```
name keyword can trigger extensions to autofill users first name